orbit/cmd/desktop/desktop.go

```diff
@@ -4,43 +4,48 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
-	"log"
 	"net/url"
 	"os"
 	"path"
+	"runtime"
 	"time"
 
 	"github.com/fleetdm/fleet/v4/pkg/open"
 	"github.com/fleetdm/fleet/v4/server/service"
 	"github.com/getlantern/systray"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 var version = "unknown"
 
 func main() {
+	setupLogs()
+
 	// Our TUF provided targets must support launching with "--help".
 	if len(os.Args) > 1 && os.Args[1] == "--help" {
 		fmt.Println("Fleet Desktop application executable")
 		return
 	}
-	log.Printf("fleet-desktop version=%s\n", version)
+	log.Info().Msgf("fleet-desktop version=%s\n", version)
 
 	devURL := os.Getenv("FLEET_DESKTOP_DEVICE_URL")
 	if devURL == "" {
-		log.Println("missing URL environment FLEET_DESKTOP_DEVICE_URL")
+		log.Fatal().Msg("missing URL environment FLEET_DESKTOP_DEVICE_URL")
```

Comment:
No need for the `os.Exit` call (same in the other places where `log.Fatal().Msg()` is called):
```go
// Fatal starts a new message with fatal level. The os.Exit(1) function
// is called by the Msg method.
```

Reply:
nice! didn't know that. Fix pushed.